src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3174,6 +3194,11 @@ SvgLoader::~SvgLoader()
 
 void SvgLoader::run(unsigned tid)
 {
+    if (((uint32_t)viewFlag & (uint32_t)SvgViewFlag::Viewbox) && (fabsf(vw) < FLT_EPSILON || fabsf(vh) < FLT_EPSILON)) {
```

Comment:
I moved the log to the header, but this check is still here, so I can return from the func. 
I don;t want to return false in a heder, since then the ```load``` api returns InvalidArguments.
is it what you meant?

Reply:
If you argue the condition is "Success" then it's ok otherwise InvalidArguments would be better or we can return some else.(you can define and suggest)

Regardless of that, there is no need to run a thread since we could know the condition prior to run() method.
Maybe you can check this at read() instead.

Reply:
I moved this check to read, now Result::Unknown is returned + log